core/vtk/ttkFTMTree/ttkFTMTreeUtils.h

```diff
@@ -0,0 +1,87 @@
+/// \ingroup vtk
+/// \class ttk::ttkFTMTreeUtils
+/// \author Mathieu Pont (mathieu.pont@lip6.fr)
+/// \date 2021.
+///
+/// Utils function for manipulating FTMTree class
+
+#ifndef _TTKFTMTREEUTILS_H
+#define _TTKFTMTREEUTILS_H
+
+#pragma once
+
+#include <FTMTree.h>
+
+#include <ttkUtils.h>
+
+#include <vtkCellData.h>
+#include <vtkIntArray.h>
+#include <vtkPointData.h>
+#include <vtkUnstructuredGrid.h>
+
+using namespace ttk;
+using namespace ftm;
+
+struct MergeTree {
```

Comment:
move to core/base/ftmTree/FTMTreeUtils ?

Reply:
Done!